controllers/equipments.js

```diff
@@ -0,0 +1,44 @@
+const Category = require('../models/category');
+const Equipment = require('../models/equipment');
+const User = require('../models/user');
+
+exports.create = async (req, res) => {
+  const equipment = await Equipment.create(req.body);
+  if (equipment.errors) {
+    const currentUser = await User.find(req.session.userId);
+    const categories = await Category.all();
+    const equipments = await Equipment.all();
+    return res.render('admin', {
```

Comment:
:( is there a better way? it sucks to render admin and have the url be a failed equipment post. Save error in session?

Reply:
Done